### PR TITLE
Added background carousel

### LINKED ISSUE #106: Adding a background carousel
Hello ! I am a GSSOC Participant 
Here rather than having a purple background can we have a background carousel with images and post opportunity button over it, I think it will look nice. Do assign me this issue Thank you

### PR DESCRIPTION
**Pull request by drishtipeshwani**
# Description

I have added an image carousel background where the post opportunity button is there.  In this manner - 

Fixed #106  (e.g. Fixed #8)

## Type of change

Please check options that are relevant to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/Components/ViewOpportunity/index.js [hunk 1]
```diff
@@ -85,28 +90,23 @@ export function ViewOpportunity(props) {
...
 
+          <Navbar />
+          <h3
+            style={{
+              textAlign: 'center',
+              marginTop: '220px',
+              marginBottom: '200px',
+            }}
+
+
```
Comment (abdus) [inline src/Components/ViewOpportunity/index.js]: Syntax Error. 

```suggestion
        <h3
          style={{
            textAlign: 'center',
            marginTop: '220px',
            marginBottom: '200px',
          }}
        >
          Loading Opportunities if any...
        </h3>
```

**Review by abdus — CHANGES_REQUESTED** [review]

**Comment by abdus** [comment]
hey @drishtipeshwani, there are some syntax errors. Fixing them would make the check pass. I tried to fix it but could not.

**Comment by drishtipeshwani** [comment]
> hey @drishtipeshwani, there are some syntax errors. Fixing them would make the check pass. I tried to fix it but could not.

I will check

**Comment by abdus** [comment]
@drishtipeshwani let me know if you are still working on this PR.

**Comment by vaishali614** [comment]
@drishtipeshwani Due to your inactiveness, we're closing this PR.

**Pull request closed without merging**